pages/solidity-api/ServicesRegistry.md

```diff
@@ -169,7 +169,11 @@ function createService(bytes32 _serviceId, struct ServicesRegistry.ServiceRecord
 Creates a new service. Services contain multiple different record types which could be a tokenUri, a URL or any other
 unstructured content. Each record is identified by its recordType. We expect off-chain standardization around recordTypes and
 do not maintain a canonical on-chain list of records. Associated with each recordType is a content string which is intended
-to be interpreted by the client. The service ID must be unique and the service records must not contain duplicate record types.
+to be interpreted by the client. 
+
+#### Requirements
+ - **The service ID must be unique**
+ - **The service records must not contain duplicate record types**
```

Comment:
seemed all these had important requirements, added a "Requirements" section to show all these have requirements that are important